components/Login/index.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
```

Comment:
With Next.js we don't need to explicitly import React, usually we only do it when we need some React hook.

Reply:
Oh dope. Didn't know! Will fix